logstash-core/lib/logstash/pipeline.rb

```diff
@@ -211,17 +212,22 @@ def start_workers
   end
 
   # Main body of what a worker thread does
-  # Repeatedly takes batches off the queu, filters, then outputs them
+  # Repeatedly takes batches off the queue, filters, then outputs them
   def worker_loop(batch_size, batch_delay)
     running = true
 
+    namespace_events = metric.namespace([:stats, :events])
```

Comment:
so you mean this will create the namespace first time we actually store something? what happen then for things with value zero?